.github/workflows/integration-package-tests.yaml

```diff
@@ -98,7 +98,7 @@ jobs:
         run: >
           pytest tests
           --numprocesses auto
-          --maxprocesses 6
+          --maxprocesses 4
```

Comment:
I was running into resource contention since each process gets its own server, so I dialed the number of process for the tests down.